### DISCUSSION TITLE
SelectPicker Group

### DISCUSSION THREAD

**Discussion by hoseinabedi** [rsuite/rsuite#3579] [Q&A] [answered]
Hey guys,
I'm using SelectPicker with Group feature, but as you can see in documents, when the drop down box opens, all the group are shown in expanded mode while I want they to be in collapsed status.
Thanks in advance for your guidance and solutions.

![image](https://github.com/rsuite/rsuite/assets/8780457/f33b3ea6-587e-4d5d-b1cb-d9d6674c56c2)

**Comment by hoseinabedi** [comment]
Huh? no answer yet? is there anyone here to help me?

**Reply by simonguo** [reply, marked as answer]
Currently not supported, but you can try the TreePicker component first.

**Reply by hoseinabedi** [reply]
Thank you, it helps a me a lot 👍
